### DISCUSSION TITLE
Yoruba translation

### DISCUSSION THREAD

**Discussion by AfolabiOlaoluwa** [fastapi/fastapi#10034] [Translations]
Welcome to the issue that coordinates the Yorùbá translation effort.

Purpose:
- Avoiding several people working on the same document at the same time.
- The first person who declares that he/she is working on a translation gets the responsibility to carry it out. If a PR seems to be stalled, we can discuss a transfer of responsibility here.

Enforcing best practices:
- For all the contributions you will make, endeavor to use diacritics. The diacritized texts determines the pronounciation in Yorùbá language.  
- Yorùbá language is very expressive and you should also follow that partern when you are not getting direct translation for the English text. 
- Defining and sharing best practices will help to avoid common mistakes and will allow faster and easier reviews.
- This interpretation tasks uses Oyo Empire dialect of Yorùbá Language, as well as Ile-Ife. 

Help and build the community:
- Do not hesitate to ask any questions regarding the Yorùbá translation effort here. The stronger the community, the more effective we will be and the more we will enjoy. 

How to contribute/review:
- Keep in mind that the easiest way to participate is to review the PRs. We need to avoid accumulating PRs waiting for review.

Willing to translate?:
- If you are not familiar with contributing to open-source projects have a look at https://github.com/firstcontributions/first-contributions.
- In any case, take a look at the documentation section related to the [contribution](https://fastapi.tiangolo.com/contributing/#development-contributing) and more precisely the part about the [documentation](https://fastapi.tiangolo.com/contributing/#docs).

Once you are decided to translate a document, make yourself known here by leaving a message here.

First Translation Commit: 
- https://github.com/tiangolo/fastapi/pull/10033. This may be key to you possible translations

Preview Page: 
- https://98f961a2.fastapitiangolo.pages.dev/yo/. This may be key to you possible translations

**Comment by tiangolo** [comment]
~There's a new translation PR to be reviewed: #10033 by @AfolabiOlaoluwa~ Good job! This is done. 🍰☕

**Reply by AfolabiOlaoluwa** [reply]
@tiangolo please proceed and merge. It's approved. Then I can pick up the next document to translate.

**Reply by AfolabiOlaoluwa** [reply]
@tiangolo Is there any other thing to do to get it merged after approval and completion? What's your contribution processes?

**Comment by AfolabiOlaoluwa** [comment]
This is to let everyone joining me on this translation tasks know that I will be proceeding to add translation for `docs/yo/docs/features.md` and make a PR.

For https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/features.md
